crates/aptos/src/account/key_rotation.rs

```diff
@@ -152,104 +251,65 @@ impl CliCommand<RotateSummary> for RotateKey {
             .await
             .map(TransactionSummary::from)?;
 
-        let string = serde_json::to_string_pretty(&txn_summary)
+        let txn_string = serde_json::to_string_pretty(&txn_summary)
             .map_err(|err| CliError::UnableToParse("transaction summary", err.to_string()))?;
 
-        eprintln!("{}", string);
-
         if let Some(txn_success) = txn_summary.success {
             if !txn_success {
+                eprintln!("{}", txn_string);
```

Comment:
I'm afraid I'm not sure what you mean by this question:

> We're not hiding important information from the user now?

However, the rationale for including the print output with the txn info here is so that the user has immediate feedback on if the rotation has succeeded or not onchain, because there could theoretically be an error later with updating the config and if this were the case I figured the user should at least know the txn went through or not

Reply:
My point is it used to be printed unconditionally, now it's not printed if the txn succeeded.

Reply:
I understand your concern now, but the txn info is *still* printed in the case of success:

https://github.com/aptos-labs/aptos-core/blob/5b9dc700eea51361fc5619c74899ffffd27d39a7/crates/aptos/src/account/key_rotation.rs#L327-L332